mmvb_backend/benchmarking_sessions/api/views.py

```diff
@@ -70,10 +76,15 @@ def benchmark_status(self, request, *args, **kwargs):
 
         strip_values_from_responses(result.info["responses"])
 
+        ais = benchmarking_session.ai_implementations.all()
+        ai_ids = [str(ai.id) for ai in ais]
+        statistics = result.info["statistics"]
+        statistics["table"] = get_stats_table(result.info["responses"], ai_ids)
+
         return Response(
             {
-                "status": BenchmarkingSession.Status.RUNNING,
-                "report": result.info,
+                "status": benchmarking_session.status,
+                "statistics": statistics,  # {'currentCaseIndex': 30, 'totalCaseCount': 50}
```

Comment:
Could you remove that comment?

Was confused why it was hardcoded until I saw it's commented out.